ci_framework/roles/reproducer/tasks/main.yml

```diff
@@ -190,15 +190,10 @@
       ansible.builtin.pip:
         requirements: https://raw.githubusercontent.com/openstack-k8s-operators/ci-framework/main/ansible-requirements.txt
 
-- name: Sync known repositories to ansible controller
-  when:
-    - cifmw_job_uri is undefined
-  delegate_to: localhost
-  ansible.builtin.command:  # noqa: command-instead-of-module
-    cmd: "rsync -ar {{ item.src }} zuul@controller-0:{{ item.dest }}"
-  loop: "{{ cifmw_reproducer_repositories }}"
-  loop_control:
-    label: "{{ item.src | basename }}"
+- name: Push local code
+  tags:
+    - bootstrap_repositories
```

Comment:
I thought you dropped tags, but it's here. However it's not called anymore `bootstrap`. What do you think about adding `bootstrap` tag back? Especially, because it's a part of the setup process:
https://github.com/openstack-k8s-operators/ci-framework/blob/f635b814b0bc9f29345c05ffff7b595ea42b6ff6/docs/source/reproducers/03-zuul.md#ansible-tags-of-interest

Reply:
yes and no - maybe. we can refine with a further iteration. I'll need to dig into the doc anyway, many things will change/be deprecated with the coming validated architecture change. I want to take the time next week for that work.

Reply:
I understand your PoV. However this is a change which changes how the code behaves. Especially with your recent "reproducer" changes.
Keeping up updates to the docs, with updates to the code allows us to spare extra cycles of the time in a future.

Reply:
Re-added bootstrap tag.
Updated docs accordingly